### PR TITLE
Fixed issue of clicking Liked Projects on user page not doing anything

### PR DESCRIPTION
**Pull request by stowellm**
Addresses #1795 

And while I was at it, I fixed our tests.  They no longer have extremely long names that overflow the PASS/FAIL/ERROR column and no longer contain deprecated window re-sizing code :+1:

### CODE REVIEW

File: test/functional/data_sets_controller_test.rb [hunk 1]
```diff
@@ -9,19 +9,19 @@ class DataSetsControllerTest < ActionController::TestCase
     @crunch = users(:crunch)
   end
 
-  test 'should redirect to viz for show data set' do
+  test 'redirect to viz for show data set' do
```
Comment (jaypoulz) [inline test/functional/data_sets_controller_test.rb]: change viz to vis while you're at it

File: app/assets/javascripts/users/show.js.coffee [hunk 2]
```diff
@@ -11,8 +11,7 @@ IS.onReady "users/show", ->
   nav_list = []
 
   ($ '#user_filter li').each (index) ->
-    if index != 0
-      nav_list.push ($ @).text()
```
Comment (stowellm) [inline app/assets/javascripts/users/show.js.coffee]: This was the culprit.  Someone *explictly* chose to not include the Liked Projects tab (index 0) in the nav_list array. :-1:

Comment (jaypoulz) [inline app/assets/javascripts/users/show.js.coffee]: Nice catch.... that's weird :rabbit:

**Comment by jaypoulz** [comment]
Will merge if passes testing

**Pull request merged**